src/components/pages/CompanyCertificates/UploadCompanyCerificate.tsx

```diff
@@ -73,13 +74,11 @@ export default function UploadCompanyCertificate({
     try {
       if (uploadedFile != null) {
         const data = {
-          certificateType:
-            selectedCertificateType != null
-              ? selectedCertificateType.certificateType
-              : '',
+          certificateType: selectedCertificateType?.certificateType ?? '',
           document: uploadedFile,
-          id: '',
+          expiryDate: date?.toISOString(),
```

Comment:
here it looks like
expiryDate: string | undefined
why not use that and define as any?